src/ranges.js

```diff
@@ -51,8 +51,8 @@ const filterRanges = function(timeRanges, predicate) {
  */
 export const findRange = function(buffered, time) {
   return filterRanges(buffered, function(start, end) {
-    return start - TIME_FUDGE_FACTOR <= time &&
-      end + TIME_FUDGE_FACTOR >= time;
+    return start - SAFE_TIME_DELTA <= time &&
```

Comment:
Since we were using `TIME_FUDGE_FACTOR` instead of `SAFE_TIME_DELTA` when comparing against current time badly aligned audio and video would cause us to `resetEverything`. Example:


currentTime = `0` 
video buffered = `[{start: 0.08, end: 6.17}]` 
audio buffered = `[{start: 0, end: 6.1}]` 


Due to how the code works the tech and loader buffered values will adopt the `video` buffered value. This means that with `TIME_FUDGE_FACTOR` 
`0.08 - (0.03333) <= 0` is false but with `SAFE_TIME_DELTA` this will be true: `0.08 - (0.1) <= 0`. 


I think that `SAFE_TIME_DELTA` is more appropriate here because of the comment for it, and seeing that it fixes this behavior. 

```
// Comparisons between time values such as current time and the end of the buffered range
// can be misleading because of precision differences or when the current media has poorly
// aligned audio and video, which can cause values to be slightly off from what you would
// expect. This value is what we consider to be safe to use in such comparisons to account
// for these scenarios.
```

Reply:
I also think it's possible that we should be using `SAFE_DELTA_TIME` in three other places, where we are comparing against `currentTime`. It might be worth looking into these in another pull request.

https://github.com/videojs/http-streaming/blob/master/src/segment-loader.js#L1144
https://github.com/videojs/http-streaming/blob/master/src/ranges.js#L70
https://github.com/videojs/http-streaming/blob/master/src/playback-watcher.js#L385

Reply:
Those other locations would require more investigation, but this specific change feels pretty safe to me